### ISSUE TITLE
Error when searching

### ISSUE THREAD

**Issue by frankMagoba** [yshean/google_places_flutter#2]
I get this error while trying to search for a location
![Screenshot from 2020-11-25 21-50-47](https://user-images.githubusercontent.com/38047577/100269988-71f64000-2f68-11eb-9edd-126040baa119.png)

**Comment by frankMagoba** [comment]
Thanks in advance for your feedback

**Comment by dospringer** [comment]
Have you added your API key?

**Comment by frankMagoba** [comment]
Yes. The error seemed to have been due to having disabled places API. It worked after I did. Thanks!

**Comment by vallipappula** [comment]
I got Same Issue

**Comment by doantrunghieu2607** [comment]
Hi I have an issue, the screen is just loading forever. I have input the API key and enabled Google Place API.

![Simulator Screen Shot - iPhone 13 - 2021-11-30 at 13 45 01](https://user-images.githubusercontent.com/26267090/144110458-a7bdd7c6-4dc9-4240-b0b5-0989dc393324.png)
